projboard/tests/test_client.py

```diff
@@ -5,3 +5,12 @@
 def test_homepage(client):
     response = client.get("/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_searchpage(client):
```

Comment:
Consider renaming test
to test_empty_searchpage_results

Why do you need the marker.db here? 
You do not have any interaction with it

Consider writing fixture with marker.db for another test asserts results against the fixture contents in db.

Reply:
How can I test this page with result ?? 
I should send get request with parameters , how can I do that ??
if you have an example ,send please.

Reply:
thanks a lot.

Reply:
done